packages/fulfillment/src/services/fulfillment-module-service.ts

```diff
@@ -113,6 +117,100 @@ export default class FulfillmentModuleService<
     return joinerConfig
   }
 
+  protected static normalizeFiltersAndConfigForShippingOptionsFiltering(
```

Comment:
Can we maybe split the filter and config normalization in two functions, and then you simply call

```
const filters = normalizeFilters(...)
const config = normalizeConfig(...)
```

Reply:
the config is dependent on what is in the filter as well, that is why I kept them together. I prefer having only one function for this case, but I like riqwan naming suggestion

Reply:
Yup the name is definitely better. Just to decouple the two a bit (if possible), can't we do
```
const filters = normalizeFilters(...)
const config = normalizeConfig(filters, ...)
```

So like that `normalizeConfig` can operate on top of normalized filters? Just thinking out loud, I'm fine with them being together if you prefer it 👍